gluonnlp/model/char_encoder.py

```diff
@@ -0,0 +1,145 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Convolutional character encoder."""
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+__all__ = ['CharacterEncoder']
+
+from mxnet import gluon, nd
+from mxnet.gluon import nn
+
+from .highway import Highway
+
+
+class CharacterEncoder(gluon.Block):
+    r"""Convolutional character encoder.
+
+    We implement the convolutional character encoder proposed in the following work::
+
```

Comment:
Would it make sense to set the default parameters according to the paper?

Reply:
I originally thought the paper has the setting only on PTB dataset, so I didn't follow the setting.  Besides, I added the description in the docs.